install_config/cluster_metrics.adoc

```diff
@@ -323,15 +323,28 @@ properly.
 ==== Using Generated Self-Signed Certificates
 
 The Metrics Deployer can accept multiple certificates using secrets. If a
-certificate is not passed as a secret, the deployer will generate a self-signed
-certificate to be used instead. For the deployer to generate certificates for
-you, a secret is still required before it can be deployed. In this case, create
-a "dummy" secret that does not specify a certificate value:
+certificate is not passed as a secret, then the deployer will generate and use a
+self-signed certificate instead, however this forces users to accept the certificate as a security exception.
+
+In order to use official certificates for the web console, you must use a
+link:../install_config/cluster_metrics.html#metrics-reencrypting-route[re-encrypting route].
+This allows the self-signed certificates to remain in use internally
+while allowing your own certificates to be used for externally access. When
+using a re-encrypting route, do not set the certificates as a secret, but a
+"dummy" secret named *metrics-deployer* must still exist for the Metrics
+Deployer to generate certificates.
```

Comment:
I'm confused about the above. It goes from a warning into an intrsuction? Maybe:

"When using a re-encrypting route, do not set the certificate as a secret. A "dummy" secret named *metrics-deployer* must still exist for the metrics deployer to generate certificates."